Src/VimCore/IncrementalSearch.fs

```diff
@@ -5,231 +5,344 @@ open Microsoft.VisualStudio.Text.Operations
 open Microsoft.VisualStudio.Text.Editor
 open Microsoft.VisualStudio.Text.Outlining

```

Comment:
For this file it may be easier to just look at the new code and ignore the diff. Practically none of the original implementation remains.